### ISSUE TITLE
doc: generate docs from docstrings in python code

### ISSUE THREAD

**Issue by jsilhan** [rh-lab-q/rpg#188]
see what sphinx supports
http://sphinx-doc.org/extensions.html#builtin-sphinx-extensions
depends on #121

**Comment by jsilhan** [comment]
You're doing well, Lukas. Here's another task. I would like to see generated documentation of classes `Base` and `Spec`. You don;t have to create docstrings with description of atributes in this task- just make sure all public methods and attributes (not strating with `_`) from these classes are there.
